utilities_common/multi_asic.py

```diff
@@ -0,0 +1,164 @@
+import argparse
+import functools
+import json
+import subprocess
+
+import click
+
+from sonic_device_util import get_all_namespaces
+from sonic_device_util import get_namespaces
+from sonic_device_util import get_num_npus
+from sonic_device_util import is_multi_npu
+from sonic_device_util import is_bgp_session_internal
+from sonic_device_util import is_port_channel_internal
+from sonic_device_util import is_port_internal
+
+
```

Comment:
You can delete these blank lines and combine these into one group, as both sonic_device_util  and swsssdk packages are local/SONiC-specific.

Reply:
Fixed in the latest commit

Reply:
click is also third-party, so it should be ordered as
```
import argparse
import functools

import click
from sonic_py_common import multi_asic
from utilities_common import constants
```